doc/user/content/cli/_index.md

```diff
@@ -200,3 +202,61 @@ etc.), and then create a new node with those items.
 [gh-feature]: https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md
 [scv]: /sql/show-create-view
 [scs]: /sql/show-create-source
+
+### Persistence
+
+{{< warning >}}
+
+Persistence is a experimental feature at this time. Please don't use it
+with any data you are not comfortable losing. To use persistence you need to have
+both the `--persistence` and `--experimental` flags enabled.
+
+{{< /warning >}}
+
+Materialize lets you persist input data from Kafka sources to files on the
+Materialize instance's local hard drive, so that subsequent restarts can read the
+previously ingested data from those local files instead of reading the data from
+Kafka again. Currently, source persistence will not solve many restart time or
+memory usage related problems, as it presently only copies data directly from
+Kafka and stores them in local files.
+
+We recommend using persistence if you are using real-time consistency Kafka sources,
+need to relieve load on upstream Kafka brokers, and are comfortable using
+[experimental](#experimental-mode) features. We will continue iterating on
+persistence and making it support a wider set of source types and improving
+performance and reliability.
+
+Materialize will store one copy of all input data for each persistent Kafka source.
```

Comment:
throughout change "will `<foo>`" to "`<foo>`s", e.g. "will store" to "stores" or "will buffer" to "buffers"

Reply:
done